src/cpp/include/openvino/genai/tokenizer.hpp

```diff
@@ -21,6 +22,7 @@ using Vocab = std::unordered_map<std::string, int64_t>;  // similar to huggingfa
 struct TokenizedInputs {
     ov::Tensor input_ids;
     ov::Tensor attention_mask;
+    std::optional<ov::Tensor> token_type_ids;  // optional tensor for token type ids, used in some models
```

Comment:
The comment says what the code already said. You can give an example of the model in the comment instead